learn/configuration/typo_tolerance.md

```diff
@@ -77,9 +77,9 @@ Meilisearch uses a prefix [Levenshtein algorithm](https://en.wikipedia.org/wiki/
 
 The [number of typos referenced above](#minwordsizefortypos) is roughly equivalent to Levenshtein distance. The Levenshtein distance between two words _M_ and _P_ can be thought of as "the minimum cost of transforming _M_ into _P_" by performing the following elementary operations on _M_:
 
-- substitution of a character (e.g., **k**itten → **s**itten)
-- insertion of a character (e.g., siting → sit**t**ing)
-- deletion of a character (e.g., satu**r**day → satuday)
+- substitution of a character (e.g., **`k`**`itten` → **`s`**`itten`)
+- insertion of a character (e.g., `siting` → `sit**t**ing`)
+- deletion of a character (e.g., `satu**r**day` → `satuday`)
```

Comment:
To me this looks a bit strange:

<img width="490" alt="Screen Shot 2022-06-07 at 1 43 49 PM" src="https://user-images.githubusercontent.com/68053732/172371348-214de04b-ece2-4202-bab5-b3a3e3519238.png">

Maybe we could just remove the bolding and trust users to find the difference between the two words?

```suggestion
- substitution of a character (e.g., `sitten` → `kitten`)
- insertion of a character (e.g., `siting` → `sitting`)
- deletion of a character (e.g., `saturrday` → `saturday`)
```